packages/genui/a2ui-playground/src/pages/ComponentsPage.css

```
@@ -659,3 +659,374 @@
+
+.openuiCompPropsTableWrap .compTableCell {
+  overflow-wrap: anywhere;
+}
+
+.openuiCompDslBlock {
+  max-height: 440px;
+  margin: 0;
+  border: 1px solid color-mix(in srgb, var(--geist-border) 78%, transparent);
+  border-radius: var(--geist-radius-md);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)